data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2012](https://gofossilfree.org/about/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Scientists for Future",
+            link: "https://www.scientists4future.org/",
+            keywords: ["science", "grassroots", "education", "ngo"],
+            label: "scientists-for-future",
+        },
+        description: {
+            content: "An interdisciplinary group of scientists working for a sustainable future. They support Fridays for Future and established [Lectures for Future](https://www.scientists4future.org/infomaterial/vortragsreihen/) to push the climate crisis onto the agenda in universities.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "Probably [2019](https://www.scientists4future.org/about/faq/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Europe Beyond Coal",
+            link: "https://www.beyond-coal.eu",
+            keywords: ["coal", "grassroots", "education", "network", "ngo"],
+            label: "europe-beyond-coal",
+        },
+        description: {
+            content: "They pressure governments, investors and insurers to stop supporting extraction and burning of coal in Europe. They track all planned and running [coal plants in the EU](https://beyond-coal.eu/data/) with data on health impact and emissions"
+            +" Europe Beyond Coal is supported by HEAL, WWF, Foundation “Development YES – Open-Pit Mines NO”, Greenpeace, CEE Bankwatch, ClientEarth and CAN Europe",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Bankwatch",
+            link: "https://bankwatch.org/",
+            keywords: ["finance", "divestment", "watchdog", "ngo"],
+            label: "bank-watch",
+        },
+        description: {
+            content: "They investigate the impacts of public finance on communities and help protect their rights and livelihoods."
+            + " Bankwatch monitors the European Bank for Reconstruction and Development, the European Investment Bank, the Asian Infrastructure Investment Bank and the EU in creating future budgets."
+            + " They meet their representatives to make local voices heard to fight for more sustainable and fair investments."
+            + " In addition Bankwatch works on the People's Budget, a proposed budget for the EU that (among others) aims at ending funding for fossil fuels.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Development YES - Open Pit Mines NO",
+            link: "https://rozwojtak-odkrywkinie.pl",
```

Comment:
Shall we put the english link?
https://rozwojtak-odkrywkinie.pl/en/

Reply:
Thought about it, I slightly prefer not putting language selectors in the links, assuming everybody can find the button if they really want to. Having a single main link per initiative slightly reduces effort when we translate this at some point. Let's change it if you feel strongly about it!

Reply:
Ok, got another opinion on that. I always assume that users are lazy and as most of humans unlikely see new stuff. A unknown language more likely shows the users that he / she is in the wrong place and has to search further. The users feels familiar in english and even if this organization is location wise not the right one, it might interest him or her.
I always would like to put the english link and only the english.
Happy to discuss :)